tests/components/xiaomi_miio/test_config_flow.py

```diff
@@ -300,6 +302,23 @@ async def test_config_flow_gateway_cloud_login_error(hass):
     assert result["step_id"] == "cloud"
     assert result["errors"] == {"base": "cloud_login_error"}
 
+    with patch(
+        "homeassistant.components.xiaomi_miio.config_flow.MiCloud.login",
+        side_effect=MiCloudAccessDenied({}),
```

Comment:
```suggestion
        side_effect=MiCloudAccessDenied(),
```
I think there is no need to pass an empty dict to the exception ctor?

Reply:
you are right

Reply:
actually no, apprently the message argument is required.